Procfile

```
@@ -0,0 +1,2 @@
+release: python manage.py migrate
+web: gunicorn authors.wsgi
```

Blank line at the end of file
```suggestion
web: gunicorn authors.wsgi

```